pytorch_lightning/utilities/imports.py

```diff
@@ -54,7 +54,7 @@ def _module_available(module_path: str) -> bool:
         return False
     try:
         importlib.import_module(module_path)
-    except ModuleNotFoundError:
+    except (ImportError, ModuleNotFoundError):
```

Comment:
nit: `ModuleNotFoundError` is a subclass of `ImportError`.
```suggestion
    except ImportError:
```